R/Spectra.R

```diff
@@ -43,15 +51,26 @@ setMethod("readMsObject", signature(object = "Spectra",
                                                  "PlainTextParam")))
                   stop("Can not read a 'Spectra' object with backend '",
                        variables["backend"], "'")
-              b <- readMsObject(object = do.call(what = variables[["backend"]],
-                                                args = list()),
-                               param = param, spectraPath = spectraPath)
+              ## Check if the library to load the backend class is available.
+              ## This should also enable backends that are defined in other
+              ## packages than Spectra. Accessing directly the "globalenv" to
+              ## ensure we can access functions/classes there.
+              fun <- getFunction(variables[["backend"]], mustFind = FALSE,
```

Comment:
crazy that completely work locally for me ! Good that we have different ecosystem

Reply:
Well. It also worked for me when running `rcmdcheck::rcmdcheck()`, but not if I was running the unit test line by line. Not sure if it's really required, but I just wanted to have some additional safety mechanism that throws an user-understandable error (with suggesting a possible solution). My thinking was that for now it all works good, because we use backends that are defined in *Spectra* - but what if we used `MsBackendSql` or `MsBackendMgf`? so, the user wants to import them?

Reply:
Ye that is completely fair I agree ! Thanks